### DISCUSSION TITLE
ImageMagick x:root

### DISCUSSION THREAD

**Discussion by bill1bailey** [ImageMagick/ImageMagick#4941] [Help]
In our app we call an ImageMagick function: 

```convert x:root fig_1.gif```

using the linux system() function. The gif file does not show the windows we created using:

 ``` shell = XtVaCreatePopupShell(name,topLevelShellWidgetClass,toplevel,
		XmNgeometry,            createGeometry(x,y),
		XmNwidth,               width,
		XmNheight,              height + getToplevelToolbarPadding(),
		NULL);
```

toplevel is the value returned from XtVaAppInitialize:

```toplevel = XtVaAppInitialize(&app,appname,NULL,0,&cargc,argv,fallback,NULL);```

This used to work but is not working in CentOS 7 or 8.

Can you help me find the problem?

**Comment by urban-warrior** [comment]
Try this command:

```
import -screen -window root fig_1.gif
```

Does that command work for you?

**Comment by bill1bailey** [comment]
I’ll try!

On Sun, Mar 13, 2022 at 6:02 PM Cristy ***@***.***> wrote:

> Try this command:
>
> import -screen -window root fig_1.gif
>
> Does that command work for you?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ImageMagick/ImageMagick/discussions/4941#discussioncomment-2352524>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT42PCY5QB6UBZ43VXIYYFTU7ZXX7ANCNFSM5QTUAJ7A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 

William I. Lundgren - President & CEO
856-231-4458 x104
www.gedae.com

"The intuitive mind is a sacred gift and the rational mind its faithful
servant. We have created a society that honors the servant and has
forgotten the gift." - A. Einstein

"The design of life is expressed in the stable states of DNA."

**Comment by bill1bailey** [comment]
Hi Cristy,

That seems to work! Thank you. I will integrate that into the application
to see if it fixes our problem.

Thank you,
Bill

On Sun, Mar 13, 2022 at 6:21 PM William Lundgren ***@***.***>
wrote:

> I’ll try!
>
> On Sun, Mar 13, 2022 at 6:02 PM Cristy ***@***.***> wrote:
>
>> Try this command:
>>
>> import -screen -window root fig_1.gif
>>
>> Does that command work for you?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/ImageMagick/ImageMagick/discussions/4941#discussioncomment-2352524>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AT42PCY5QB6UBZ43VXIYYFTU7ZXX7ANCNFSM5QTUAJ7A>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***
>> com>
>>
> --
>
> William I. Lundgren - President & CEO
> 856-231-4458 x104
> www.gedae.com
>
> "The intuitive mind is a sacred gift and the rational mind its faithful
> servant. We have created a society that honors the servant and has
> forgotten the gift." - A. Einstein
>
> "The design of life is expressed in the stable states of DNA."
>


-- 

William I. Lundgren - President & CEO
856-231-4458 x104
www.gedae.com

"The intuitive mind is a sacred gift and the rational mind its faithful
servant. We have created a society that honors the servant and has
forgotten the gift." - A. Einstein

"The design of life is expressed in the stable states of DNA."

**Comment by bill1bailey** [comment]
At the suggestion of Cristy from ImageMagick, this worked:

import -screen -window root -crop 200x400+400+200 fig_1.gif
